pystan/experimental/pickling_tools.py

```diff
@@ -154,7 +168,7 @@ def get_module_name(path, open_func=None, open_kwargs=None):
     module_name = ""
     with open_func(path, **open_kwargs) as f:
         # scan first few bytes
-        for i in range(500):
+        for i in range(10000):
```

Comment:
Does this fix some bug?

Reply:
if user saves stan str model first. It is not perfect, but user can add the name manually